analytic_tag_dimension/models/account_analytic_tag.py

```diff
@@ -50,12 +50,17 @@ def write(self, vals):
                         raise ValidationError(
                             _(
                                 "You can not set two tags from same dimension.\n"
-                                " Records {} in the model {} have {}".format(
-                                    records_to_update.ids,
-                                    model._description,
-                                    same_dimension_tags.mapped("display_name"),
-                                )
+                                " Records %(records_to_update)s in the model "
+                                "%(description)s have %(display_name)s"
```

Comment:
Is this change due to translation issues?

Reply:
yes, and pre-comit and @pedrobaeza comments